crates/gpui/build.rs

```diff
@@ -21,6 +21,25 @@ fn main() {
     #[cfg(all(target_os = "macos", not(feature = "macos-blade")))]
     #[cfg(not(feature = "runtime_shaders"))]
     compile_metal_shaders(&header_path);
+
+    // We are using Windows Visita+, right..?
```

Comment:
```suggestion
    // We are using Windows Vista+, right..?
```

Side note: The `rust-toolchain.toml` file mandates the Rust version of `1.76`, which is incidentally the first version to restrict official Windows support to [Windows 10 and Windows 11](https://github.com/rust-lang/compiler-team/issues/651).

Reply:
This actually is news to me. Additionally, the `alacritty_terminal` dependency that Zed relies on is implemented through conPTY API on Windows, while this API is only supported starting from the Windows 10 Fall 2018 update. So, I guess these users will also be left behind..?